collectors/python.d.plugin/edgex/edgex.chart.py

```diff
@@ -0,0 +1,312 @@
+update_every = 5
+
+CHARTS = {
+    'events_throughput':{
+        'options': [None, 'Events and Readings (readings are a certain type of event) per second in the EdgeX platform', 'events/s',
+                    'events throughput','edgex.events', 'line'],
+        'lines': [
+            ['events', None, 'incremental'],
+            ['readings', None, 'incremental']
+        ]
+    },
+    # 'readings_throughput':{
+    #     'options': [None, 'Readings per second in the EdgeX platform', 'readings/s', 'throughput', 'edgex.readings',
+    #                 'line'],
+    #     'lines': [
+    #         ['readings', None, 'incremental']
+    #     ]
+    # },
+    'events_count':{
+        'options': [None, 'Total number of events and readings in the EdgeX platform', 'events', 'events count', 'edgex.events_readings_abs',
+                    'line'],
+        'lines': [
+            ['readings', None, 'absolute'],
+            ['events', None, 'absolute']
+        ]
+    },
+    'devices_number':{
+        'options': [None, 'Number of registered devices in the EdgeX platform', 'devices', 'devices count', 'edgex.devices_number',
+                    'line'],
+        'lines': [
+            ['registered_devices', None, 'absolute'],
+        ]
+    },
+    'memory_alloc':{
+        'options': [None, 'Alloc: currently allocated number of bytes on the heap of each EdgeX service. Provided by the Golang runtime Package.',
+                    'bytes', 'memory metrics', 'edgex.memory_alloc', 'line'],
+        'lines': [
+            ['core_data_alloc', None, 'absolute'],
+            ['core_metadata_alloc', None, 'absolute'],
+            ['core_command_alloc', None, 'absolute'],
+            ['support_logging_alloc', None, 'absolute']
+        ]
+    },
+    'memory_malloc':{
+        'options': [None, 'Mallocs: it is the cumulative count of heap objects allocated. Provided by the Golang runtime Package.', 'heap objects',
+                    'memory metrics','edgex.memory_malloc', 'line'],
+        'lines': [
+            ['core_data_malloc', None, 'absolute'],
+            ['core_metadata_malloc', None, 'absolute'],
+            ['core_command_malloc', None, 'absolute'],
+            ['support_logging_malloc', None, 'absolute']
+        ]
+    },
+    'memory_frees':{
+        'options': [None, 'Frees: it is the cumulative count of heap objects freed. Provided by the Golang runtime Package.', 'heap objects',
+                    'memory metrics', 'edgex.memory_frees', 'line'],
+        'lines': [
+            ['core_data_frees', None, 'absolute'],
+            ['core_metadata_frees', None, 'absolute'],
+            ['core_command_frees', None, 'absolute'],
+            ['support_logging_frees', None, 'absolute']
+        ]
+    },
+    'memory_liveObjects':{
+        'options': [None, 'LiveObjects: the number of live objects is (Mallocs - Frees) . Provided by the Golang runtime Package.', 'heap objects',
```

Comment:
~~Provided by the Golang runtime Package.~~

Reply:
Can you please elaborate? I think that the user would want to know how the metrics are extracted (to decide on it's correctness)

Reply:
Probably you are right, but for me it looks like excessive details. 

> Provided by the Golang runtime Package

Doesnt explicitly says that it is not trustable source or something. It is just a fact. I would suggest to move it to the dashboard info

Reply:
Agreed

Reply:
@OdysLam as you may noticed chart titles are very hard to read - small font. If you want to add some details add them to the dashboard_info.js, it is more readable and their you can use markdown, links, etc

Reply:
Will move all the descriptions to the dashboard_info.js

btw, do you have any idea Netdata client keeps replacing my own custom dashboard_info.js with the EdgeX icon with a "default?" one that doesn't have them?

Reply:
Probably you have enabled autoupdate and dashboard_info.js is being overwritten on every update (once a day)